R/utils.R

```diff
@@ -12,12 +12,12 @@
 
 catalogue_base_url <- function() {
   getOption("bcdata.catalogue_gui_url",
-            default = "https://catalogue.data.gov.bc.ca/")
+            default = "https://toyger.data.gov.bc.ca/")
```

Comment:
You can do this in https://github.com/bcgov/bcdata/blob/f9ac85198ddecd92ae3c1dc7a155436cac1d5090/tests/testthat/setup.R#L23

Reply:
Nice. But this shouldn't make a different, despite it being a cleaner way to do this right?

Reply:
Oh yeah, totally fine!